api/envoy/extensions/compression/zstd/compressor/v3/zstd.proto

```diff
@@ -34,25 +34,29 @@ message Zstd {
     BTULTRA2 = 9;
   }
 
-  // Set compression parameters according to pre-defined cLevel table.
+  // Set compression parameters according to pre-defined compression level table.
   // Note that exact compression parameters are dynamically determined,
-  // depending on both compression level and srcSize (when known).
+  // depending on both compression level and source content size (when known).
   // Value 0 means default, and default level is 3.
   // Setting a level does not automatically set all other compression parameters
   // to default. Setting this will however eventually dynamically impact the compression
   // parameters which have not been manually set. The manually set
   // ones will 'stick'.
   google.protobuf.UInt32Value compression_level = 1;
 
-  // A 32-bits checksum of content is written at end of frame (default:0)
+  // A 32-bits checksum of content is written at end of frame, If not set, defaults to false.
```

Comment:
i think this should be a full-stop rather than comma before the sentence about defaults